### DISCUSSION TITLE
Add warning in editor when visibility screen notifier's is disabled

### DISCUSSION THREAD

**Discussion by EFrion** [godotengine/godot-proposals#7967] [Editor]
When the VisibleOnScreenEnabler3D is invisible, RigidBody3D and AnimationPlayer3D attached to the nodes are disabled (see the documentation [here](https://docs.godotengine.org/en/stable/classes/class_visibleonscreenenabler3d.html)). This can be confusing for new devs. Adding a warning in the editor would avoid confusion.

**Comment by theraot** [comment]
I imagine this would have to be a configuration warning on the node, as no script is involved.

I'd rather those warning be reserved for something potentially not working as intended, instead of something working as intended but the intended way being obscure.

Which make me think a "configuration info" would be more appropriate. Sadly there isn't something like that in Godot currently.

**Reply by Mickeon** [reply]
A bit noteworthy. This is a relatively new behavior. In 3.x all these nodes cared about was whether or not they were culled off-screen, the `visible` property did not matter. I can see why this is confusing, but it sounds primarily like a documentation issue (unlike the 2D verison, VisibleOnScreenEnabler3D does explain what "visible" means).
